src/Components/Alert/_alert.scss

```diff
@@ -56,21 +57,41 @@
   text-shadow: 0 1px 2px rgb(37, 34, 34);
 }
 
-@media screen and (max-width: 651px) {
+@media screen and (max-width: 1025px) {
   .alert-title {
     font-size: 2.5rem;
   }
 
   .alert-message {
-    font-size: 1.6rem;
+    font-size: 1.5rem;
+  }
+}
+
+@media screen and (max-width: 930px) {
+  .alert-title {
+    font-size: 2.3rem;
+  }
+
+  .alert-message {
+    font-size: 1.5rem;
+  }
+}
+
+@media screen and (max-width: 651px) {
+  .alert-title {
```

Comment:
Looking forward to getting Alex some exercise today. Booking a ton of walks to test 😂